src/js/pages/attendee/NewsFeed.jsx

```diff
@@ -76,17 +84,27 @@ export default class NewsFeed extends React.Component {
     alert(NewsFeedStore.error);
   }
 
+  loadPreferences = () => {
+    this.setState({ preferences: PreferenceStore.preferences });
+  }
+
+  showPreferenceError = () => {
+    alert(PreferenceStore.error);
+  }
+
   render() {
     const {
       contentFeed,
       fbIntegration,
       imageOverlay,
       twIntegration,
+      preferences,
     } = this.state;
 
     const NewsFeedPosts = contentFeed.items.map(post =>
       <NewsFeedPost
         key={post.timestamp}
+        repost={preferences.reposts}
```

Comment:
Should this be here or attached to the component that uses it? If it should be passed in from the base page, we should probably do the same for CreatePost